### PR TITLE
fix(chat): add back to chat button on marketplace mobile (Issue #2243)

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

Need to add arrow "Back to chat" to header for mobile view

Issues:

- Issue #2243 

**UI changes**

Снимок экрана 2024-10-23 в 15 37 10
Снимок экрана 2024-10-23 в 15 37 20

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

File: apps/chat/src/components/Header/BackToChatMobile.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+export const BackToChatMobile = () => {
+  const { t } = useTranslation(Translation.Header);
+
+  const router = useRouter();
+
+  return (
+    <Tooltip isTriggerClickable tooltip={t('Back to Chat')}>
+      <button
+        className="flex h-full items-center justify-center border-r border-tertiary px-2 md:px-3 lg:hidden"
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Header/BackToChatMobile.tsx]: lg it's still tablet demension. Use xl: (desktop) or md: (for tablet and desktop)

File: apps/chat/src/components/Header/BackToChatMobile.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+import Tooltip from '../Common/Tooltip';
+
+export const BackToChatMobile = () => {
+  const { t } = useTranslation(Translation.Header);
+
+  const router = useRouter();
+
+  return (
+    <Tooltip isTriggerClickable tooltip={t('Back to Chat')}>
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Header/BackToChatMobile.tsx:16]: do we need tooltip here?

File: apps/chat/src/components/Header/BackToChatMobile.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      <button
+        className="flex h-full items-center justify-center border-r border-tertiary px-2 md:px-3 lg:hidden"
+        onClick={() => {
+          router.push('/');
+        }}
+      >
+        <div className="flex items-center justify-center rounded p-[3px]">
+          <IconArrowLeft
+            className="text-secondary hover:text-accent-secondary"
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Header/BackToChatMobile.tsx:25]: maybe we need to add shrink-0 here

File: apps/chat/src/components/Header/BackToChatMobile.tsx (hunk 2 above)
Comment (IlyaBondar) [reply apps/chat/src/components/Header/BackToChatMobile.tsx:16]: @Alexander-Kezik , I suppose so, because we have collapsed view

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2437.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Header/BackToChatMobile.tsx (hunk 1 above)
Comment (Derikyan) [reply apps/chat/src/components/Header/BackToChatMobile.tsx]: I thought to show the button until the side panel appears, but I checked again according to the design, there is no button on the tablet

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2437.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2437.nightly-test.deltixhub.io
>E2E tests status: success

**Review by Alexander-Kezik — APPROVED** [review]

**Pull request merged**